homeassistant/components/frontend/__init__.py

```diff
@@ -668,6 +668,7 @@ def websocket_get_themes(
         vol.Required("language"): str,
         vol.Required("category"): str,
         vol.Optional("integration"): str,
+        vol.Optional("integrations"): [str],
```

Comment:
Let's do `vol.All(cv.ensure_list, [str])` as it's more in line with other places we allow list or single item.

Reply:
1b3577660519464d18fe97066b927cf41ba2c859